src/core/packages/http/server/src/versioning/types.ts

```diff
@@ -34,9 +34,17 @@ export type VersionedRouteConfig<Method extends RouteMethod> = Omit<
   RouteConfig<unknown, unknown, unknown, Method>,
   'validate' | 'options'
 > & {
-  options?: Omit<
+  options?: Pick<
```

Comment:
Not needed for this PR, but converted this type subset selection from blocklist to allowlist for stricter control of the interface we are exposing.